django_netjsonconfig/base/device.py

```diff
@@ -105,3 +109,10 @@ def get_default_templates(self):
     @classmethod
     def get_config_model(cls):
         return cls._meta.get_field('config').related_model
+
+
+# edit validators
+name_field = AbstractDevice._meta.get_field('name')
+name_field.validators = name_field.validators[:] + [
```

Comment:
have you understood what's happening here? Especially the `[:]` syntax? If not, please ask!